### ISSUE TITLE
PID: 227028

### ISSUE THREAD

**Issue by electricsidecar-dev** [ElectricSidecar/Acura-TLX#3]
Header: `ATSHDA60F1`

Example response:

```
18DAF1601039627028F8FC00
18DAF1602100000004040000
18DAF1602200000000000E00
18DAF160230B07FF00000000
18DAF1602400000000000000
18DAF1602500000000000000
18DAF1602600000000000000
18DAF1602700000000000000
18DAF1602800005555555555
```

https://www.ridgelineownersclub.com/threads/custom-pids-for-odb-scantool-app-like-torque-or-carscanner.229500/ indicates that Signed(Q,R) might be `Outside Air Temp`. QR in the example are `000B` -> 11, but need to confirm this is the actual temperature from when this reading happened.

**Comment by electricsidecar-dev** [comment]
[@sidbmw](https://github.com/sidbmw) to confirm the above and share any more readouts!

**Comment by sidbmw** [comment]
Yep it's 8C outside with a peak of 11C today so this seems correct. I was in the garage when I sent you the logs so a bit warmer than 8C makes sense.

**Comment by electricsidecar-dev** [comment]
`SHA: 1e0c347` merged in if you want to give it a try! Also it would be helpful to test if your dashboard settings do actually affect this parameter. If you're able to change to Fahrenheit somehow and then try reading the value again. I'm hoping it's always outputting in Celsius but this parameter might be reading straight from your dash, which will make it a bit tricky for Sidecar to know how to treat the value 🤔

**Comment by sidbmw** [comment]
Will do. Pretty sure there isn’t a way to change units in this car. I spent a good amount of time trying to change the tire pressure unit to psi from kPa 😅. Do all temp related units assume Celsius? I remember seeing exhaust related values over 700C which I don’t think is accurate. Makes more sense in Fahrenheit. 

Based on the VIN isn’t it possible to figure out where the car was sold/made for? With that maybe you can set the default temperature units. Using gps location and/or phone region could work too I think but you’d have the edge case of imported cars?

**Comment by sidbmw** [comment]
Just edited the above comment and it disappeared from the app. Bug?

**Comment by electricsidecar-dev** [comment]
Oh, odd, I see the comment in Sidecar; is it still not showing for you?

**Comment by sidbmw** [comment]
I see it now

**Comment by sidbmw** [comment]
Ok can confirm the temp shown in the app aligns with what the car shows (18C in this case). Sent you logs as well.

**Comment by jverkoey** [comment]
Awesome :) We can leave this issue open in case we ever figure out what the other bytes in this parameter mean.
